examples/routing_service_shapes_processor/cpp/README.rst

```diff
@@ -21,6 +21,7 @@ From the directory containing the example sources:
     cd build
     cmake -DCONNEXTDDS_DIR=<Connext directory> \
             -DCONNEXTDDS_ARCH=<ARCH> \
+            -DCMAKE_RUNTIME_OUTPUT_DIRECTORY
```

Comment:
This needs a value and is something you  can set as a target property

Reply:
Good catch, I forgot to update this properly. Will this be enough for Visual Studio?